test/smokes/pylint/expectations.rb

```diff
@@ -0,0 +1,152 @@
+s = Runners::Testing::Smoke
+
+s.add_test(
+  "success",
+  type: "success",
+  issues: [],
+  analyzer: { name: "Pylint", version: "2.5.2" }
+)
+
+s.add_test(
+  "failure",
+  timestamp: :_,
```

Comment:
[nitpick] This can be omitted:

```suggestion
  timestamp: :_,
```